README.md

```diff
@@ -20,6 +20,10 @@ y: |`md
 `|
 ```
 
+## Install
+
+You can install using the extension sidebar or by going to the [Visual Studio Marketplace Website](https://marketplace.visualstudio.com/items?itemName=terrastruct.d2) and clicking 'Install'.
```

Comment:
sorry I should've been more clear. I meant replacing `'Install'` with `"Install"`, though I like the installation -> install change too